nipype/interfaces/afni/preprocess.py

```diff
@@ -353,12 +353,13 @@ class AllineateInputSpec(AFNICommandInputSpec):
         argstr='-nomask',
         desc='Don\'t compute the autoweight/mask; if -weight is not '
              'also used, then every voxel will be counted equally.')
-    weight_file = File(
```

Comment:
AFNI Allineate has been there for a long while. Would it make sense to leave the `weight_file` input here with the deprecated meta set?
```Python
    weight_file = File(
        argstr='-weight %s',
        exists=True,
        deprecated='1.0.0', new_name='weight',
        desc='Set the weighting for each voxel in the base dataset; '
             'larger weights mean that voxel count more in the cost function. '
             'Must be defined on the same grid as the base dataset')
```

Reply:
>Would it make sense to leave the weight_file input here with the deprecated meta set?

Ok, sorry.